### ISSUE TITLE
bin/ does not get populated with any files

### ISSUE THREAD

**Issue by aioue** [cobbzilla/merge-yml#6]
#### Description

`bin/` is empty after compile. No script exists to run.
#### Command

`sudo mvn -X -P uberjar package`
#### Platform

```
tpai@tp-mbp:~/github/merge-yml$ mvn --version
Apache Maven 3.3.3 (7994120775791599e205a5524ec3e0dfe41d4a06; 2015-04-22T12:57:37+01:00)
Maven home: /usr/local/Cellar/maven/3.3.3/libexec
Java version: 1.8.0_45, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_45.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.10.3", arch: "x86_64", family: "mac"
```
#### Output

https://gist.github.com/aioue/3e34a4773cbc808b71f0#file-output-txt

**Comment by cobbzilla** [comment]
Dup of #3 -- it's an easy fix to add the scripts, I'd welcome a pull request.

**Comment by aioue** [comment]
This isn't a dup of #3. If I create the bin dir, the scripts are still not populated. I have submitted a PR to bring the bin/ dir back.

**Comment by cobbzilla** [comment]
@aioue You're right. I just looked a little deeper, and the bin dir gets nuked. A friend was working with the code and had added some cool stuff to auto-magically manage the bin dir, but it looks like that hasn't been working, for, well, a very long time. I just pushed a commit that should fix this and #3.

**Comment by cobbzilla** [comment]
sorry for the late reply. I've confirmed that after a fresh checkout and `mvn -X -P uberjar package`, the `bin` directory is still there and contains the `merge-yml.sh` script
